eng/scripts/Build_Perf.ps1

```diff
@@ -0,0 +1,29 @@
+#Requires -Version 7.0
+
+Param(
+    [string] $serviceDirectory
+)
+
+Push-Location sdk/$serviceDirectory
+
+if (Test-Path -Path testdata/perf) {
+    Write-Host "##[command] Building and vetting performance tests in sdk/$serviceDirectory/testdata/perf"
+
+    Write-Host "##[command] Executing 'go build .' in sdk/$serviceDirectory/testdata/perf"
+    go build .
```

Comment:
The go build will run in the service directory root. Is this intended or did you want another push location here into testdata/perf?